android/autodispose-android/src/main/java/com/uber/autodispose/android/internal/AutoDisposeAndroidUtil.java

```diff
@@ -34,14 +32,9 @@
     }
   };
 
-  private AutoDisposeAndroidUtil() { }
+  private AutoDisposeAndroidUtil() {}
 
   public static boolean isMainThread() {
     return AutoDisposeAndroidPlugins.onCheckMainThread(MAIN_THREAD_CHECK);
   }
-
-  public static boolean isAttached(View view) {
```

Comment:
why remove this? the API check abstraction is nice. also, why not use `ViewCompat#isAttachedToWindow(View)`?

Reply:
We don't have a support library dependency in this artifact to keep things lean

Reply:
Also removed just to have one less method. It was only extracted before because it was used twice